app/Models/Post.php

```diff
@@ -53,7 +53,22 @@ public function signup()
      */
     public function reactions()
     {
-        return $this->hasMany(Reaction::class);
+        // return $this->morphToMany(Reaction::class, 'postable');
+        return $this->morphMany(Reaction::class, 'postable');
+    }
```

Comment:
> I wanted to check with you to see if this seems reasonable to you before going down a rabbit hole?

Yes! I think you're 100% on the right track. Are you getting the `array_key_exists` error if you switch to plain one-to-one "morph"? It looks like that's happening in Eloquent's `getAttribute` method, so it'd make sense if it was just because it was looking for a nonexistent column.